contracts/staking/BaseShareField.sol

```diff
@@ -180,7 +182,7 @@ contract BaseShareField is DAOContract{
     // External function call
     // When user calls this function, it will calculate how many token will mint to user from his productivity * time
```

Comment:
user doesnt call this function, its fundmanager only
and it doesnt only calcualte it actually updates the user record, marking it received outstanding rewards

Reply:
Changed description